docs/integrations/microsoft-azure/web-apps.md

```diff
@@ -39,44 +52,141 @@ _sourceCategory=Azure/Web-app
-In Step 1, you create an HTTP source. When you configure the, plan your source category to ease the querying process. A hierarchical approach allows you to make use of wildcards. For example: `Azure/WebApp/Metrics`
-2. Push metrics from Azure Monitor to Event Hub.
-    1. Sign in to [Azure Portal](https://portal.azure.com/).
-    1. Go to your Azure Web App and in the left pane, go to **Monitoring** > **Diagnostics Settings.**
-    1. Diagnostic Settings blade will show all your existing settings if any already exist. Click **Edit Setting** if you want to change your existing settings, or click **Add diagnostic setting** to add a new one.
-    1. Select the **Stream to an event hub box** checkbox.
-    1. Select an Azure subscription.
-    1. **Event hub namespace.** Namespace created in [Collect Metrics from Azure Monitor](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor) by Metrics ARM template starting with `SumoMetricsNamespace<unique suffix>`.
-    1. **Event hub name (optional).** Select **insights-metrics-pt1m** from the **Select Event hub name** dropdown.
-    1. **Event hub policy.** Select **RootManageSharedAccessKey** from **Select Event hub policy name** dropdown.
-    1. Select the checkbox for **AllMetrics** types under **Metrics** which you want to ingest.
-    1. Click **Save**.
+## Setup
+
+Azure service sends monitoring data to Azure Monitor, which can then [stream data to Eventhub](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/stream-monitoring-data-event-hubs). Sumo Logic supports:
+
+* Logs collection from [Azure Monitor](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-get-started) using our [Azure Event Hubs source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
+* Activity Logs collection from [Azure Monitor](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-get-started) using our [Azure Event Hubs source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/). It is recommended to create a separate source for activity logs. If you are already collecting these logs, you can skip this step.
+* Metrics collection using our [HTTP Logs and Metrics source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/) via Azure Functions deployed using the ARM template.
+
+You must explicitly enable diagnostic settings for each storage service (blob,queue,table and file) and each storage account that you want to monitor. You can forward logs to the same event hub provided they satisfy the limitations and permissions as described [here](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings?tabs=portal#destination-limitations).
```

Comment:
storage service (blob,queue,table and file) and each storage account -> web app

Reply:
Done